src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -805,7 +805,8 @@ public EmoteImpl createEmote(GuildImpl guildObj, DataObject json, boolean fake)
         return emoteObj
                 .setName(json.getString("name", ""))
                 .setAnimated(json.getBoolean("animated"))
-                .setManaged(json.getBoolean("managed"));
+                .setManaged(json.getBoolean("managed"))
+                .setAvailable(json.getBoolean("available"));
```

Comment:
`getBoolean` returns `false` if there is no value for the key. Is it correct, that this would be false if `available` is not present in the emote json?

Reply:
I think it's.... weird. The docs don't give a clear indication about what the default value here would be. My guess is, that it would be `true` by default as Emotes added below 50 should always be available, no matter the Boost level and the available flag is probably only used in emotes that go beyond 50.
I'll quickly ask for clarification on that.

Reply:
You can pass a default value as second parameter like `json.getBoolean("available", true)`

Reply:
I'm aware of this, but before I do this would I first have a clear answer about how the missing field should be treated. Because knowing Discord could this also mean that the Emote isn't available...

Furthermore, would I need to look into adding a new event that would trigger when the Emote's availability changes, which from what I understand is tied to the Boost tier changing, but is also not reliable according to [this response here](https://github.com/discord/discord-api-docs/issues/1725#issuecomment-640947828)

Reply:
If it isn't reliable we can probably just not add an event for it. It's better to not add one now and add one later if necessary because then it doesn't involve a breaking change.

Reply:
I just pass true as default because I believe that's what Discord would say is the default for the missing field.